### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by ajskateboarder** [rokcoder-scratch/scratch-scraper#1]
For anyone willing to use the program, documentation would be helpful. You could say which modules need to be installed.

**Comment by Wjplatformer** [comment]
agreed but @RokCoder doesn't seem to be too active in GitHub

**Comment by RokCoder** [comment]
It's a fair request and I'll likely do it once I've refreshed myself with the code. I actually use GitHub on an almost daily basis but usually in private projects.

**Comment by Wjplatformer** [comment]
Yeah, to be honest, you seem to be very secretive on your GitHub activities @RokCoder , also, I can help you with the documentation for le people if you want (but might take a few weeks/months). Also, you ARE active, heh.
